sideloader/postinstall.sh

```diff
@@ -9,3 +9,6 @@ $manage compress
 
 # compile i18n strings
 $manage compilemessages
+
+# Update the search index
+$manage update_index
```

Comment:
Hmm... It think we need to look at the cron tasks in puppet to ensure we add them.